doc/images/images.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="Electron" modified="2021-07-04T13:20:10.130Z" agent="5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) draw.io/13.9.9 Chrome/85.0.4183.121 Electron/10.1.5 Safari/537.36" etag="zw_E4SXihAZpmGV92izr" version="13.9.9" type="device" pages="2"><diagram id="4zjcj5m0yfpWWFi_mls9" name="General flow">5Vldb9owFP01SNvDUGLjAI+F0W1Vq6G10p5NYhJrThw55qP99bMTJyQxjK4LYwyQUHJtX8fn3uN7YnpwGm8/CZxGDzwgrAecYNuDH3sADOFY/WrDc2FACBaGUNCgMLk7wyN9IcboGOuKBiRrdJScM0nTptHnSUJ82bBhIfim2W3JWXPWFIfEMjz6mNnW7zSQkbG63njX8JnQMDJTj8CwaIhx2dmsJItwwDc1E5z14FRwLoureDslTGNX4lKMuz3QWj2YIIl8zYDbu2+L+PsD87/Ov23uvvizxfbuw6DwssZsZRZsHlY+lwjgLC2AXdItUb4mKaeJJGK2VhNrdF1lqxbnqJsAZ5Humd9EMmZlJyn4DzLljAtlSXiiZpgwvCBszjMqKU+U2SfauWpYEyGpCsR9q8OCS8njWocbRkPdIHmqrNjcVX6WlLFyzh6AjuOORh+LR071EuNtqDO3H2c+Jv18VCpoRvoB91dxDu5E8FUS5EvS6zCYqenJ9mAw3CrEihqEx0SKZ9XFDACOyQpDCzAy95tdkg2MKaqlFzI2bNI6rDzvIq8uTPB/IxHQ8UQgSXCjGaVjwLj/Y19wS45AZVH9byljDdBIYNHtKGQ1SNAvIBGEYUnXTff7cDIzzHUa1yLiNSMC3RbUGV8Jn5hRdZ61HFUDDzmSWIREWo7ysFXLfnskPSuSn4h+vIwwxWPNoXZgVRrLZjAtErW5FtMg0MMngmT0BS9yV065OeQLRZMe0jzDK6nIm+/rHZHHdY+TZ7wnU8CpyDO8dvK4g47I447OS56RFcnZVgqs6h/wmCbJQqirUF/lRTDlGjeeZBdPKtAKIRi+jlTwVKQqI1+LxZc65L2qhtZQbxbqTUQleUyxr1s3qso3A9JSBkukv5ZOUS1e/tEjeCJr9uLTDf4Qvg3/k21qrnsUf7uU/D/4Q3Bu/MFvVRUjpyvt7Tax1vY5lip+SW4BDrTqDriASgOHg/6oVSJQH72t2tjOwHjQdnbiiuPCaxcPg67EAzqz8nbtt+mpIFiS/HwiThVDkz1Cwo9wEuo+754e31+8jhiAVh3zkLWPVgL+7wiJV7zbtg8DFmmc9BnnrYrFV5LRRJWm8rDJsQ4HDm2pfwYqQk1QoWuBuu+4AA5Ohan9llml+uenh/vDWX7p6d2WCfDcMtl++ZyWG0rH+ni5BL6/T58F3sJDXjcAo39OB9vvhBXAHQvgswB8fqE7Pr4/X5/QRV0KXdvZ3xe6Zf5cr9D1uhK63vi8QhfYRwM3aco0kkWZv/zjMK8tY09XhtTt7l/AIkS7v1Lh7Cc=</diagram><diagram id="F2rX6DZ1BJmENjO8cr3y" name="Page-2">lZG7DoIwFIafpiMJtgZwRrzEsMjg3LS1bVI4pFZBn14MRWxcdOrpd/5zRySv+62lrSqBC4NwzHtE1gjjRZzi4XmR+0iy2ANpNfeiGVT6IaZIT6+ai0sgdADG6TaEDJpGMBcwai10oewMJqzaUim+QMWo+aYnzZ3ydJGsZsdOaKl86Qyno6Omk9hPclGUQ/eBSIFIbgHcaNV9LsxredNelsU+OZ1LJpctO9A1jY43E43JNv+EvEewonG/ph6MubXhExyYFE8=</diagram></mxfile>
```

Comment:
As draw.io supports adding the editing information into the PNG file itself, it's not necessary to check in a separate `*.drawio` file. So I'd suggest removing this file. 

Just make sure that you enable the "Include a copy of my diagram" checkbox and you can simply open the PNG in draw.io:

<img src="https://user-images.githubusercontent.com/12248527/125191000-798cff80-e240-11eb-87ab-776efacbb19a.png" width="400px"/>

Reply:
Fixed it.